backend/WebApplication1/Models/Timestamp.cs

```diff
@@ -1,23 +1,33 @@
 ﻿using System;
+using WebApplication1.Infrastructure.Attributes.Database;
+using WebApplication1.Infrastructure.Extensions;
 using WebApplication1.Models.Abstract;
 
 namespace WebApplication1.Models
 {
     public class Timestamp : Model
     {
-        public string IdentifiedEntity { get; set; }
-        public int IdentifiedEntityID { get; set; }
+        public int? PersonID { get; set; }
```

Comment:
good nullable type fix, way less ugly than using 0, or any other arbitrary value, as the meaning is not the same.